.github/workflows/dockerimage-build.yml

```diff
@@ -0,0 +1,16 @@
+name: Docker Image CI
+
+on: [push]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build the Docker image
+      run: | 
+        docker build . --file docker/Dockerfile --tag devito_img
+        docker run --name testrun devito_img
```

Comment:
If i remeber correctly this just starts jupyter so gonna hang forever
it think either run tests or examples (or print_defaults if want it really quick) 

`docker run --name testrun devito_im tests`